foodji_api/Domain/Recipes/Recipe.cs

```diff
@@ -63,6 +63,17 @@ public class Recipe
         return new Recipe(name, DateTime.Now, category, description, details, ingredients, steps, imageUri, authorId);
     }
     
+    public void Update(Recipe recipe)
```

Comment:
Funky, la création entière d'une recette pour faire l'update, plutôt que de se faire un mapping différent / utiliser les propriétés directe, ou un DTO, but I'll allow it. For now, I don't see an immediate issue sur les checks faits à la création d'une recette qu'il serait mauvais de refaire, et c'est pas l'évaluation superflu d'extra d'un DateTime.Now qui va plomber la performance de l'application.

Reply:
En fait, la création de la recette (faite dans le mapper, on s'entend) valide absolument rien, toutes les validations sont implicites dans le dto. J'avais pas trop pensé comme toi là-dessus, mais j'avoue qu'on pourrait juste ne pas utiliser le mapper et passer les properties du DTO en paramètre dans le update sans créer de nouvel objet. Je vais ajouter ça au prochain commit!